File: Python/PRAK205-2210817310001-MuhammadRyanRizkyRahmadi.py
import math
st = int(input("Masukkan Sisi Tinggi="))
print, st
sm = int(input("Masukkan Sisi Miring="))
print, sm

als = math.sqrt(pow((sm), 2)- pow((st), 2))
Kel = st+sm+als
l = st*als/2
print("Alas =", round(als), "Cm")
print("Tinggi =", round(st), "Cm")
print("Keliling =", round(Kel), "Cm")
print("Luas =", round(l), "Cm^2")

File: C/PRAK205-2210817310001-MuhammadRyanRizkyRahmadi.c
#include <stdio.h>
#include <math.h>
    int main(){
        int st, sm;
            printf(" Masukkan sisi tinggi :");
            scanf("%d", &st);
            printf(" Masukkan sisi miring :");
            scanf("%d", &sm);
                int als = sqrt(pow((sm), 2)- pow((st), 2));
                int Kel = st+sm+als;
                int L = st*als/2;
                    printf("Alas = %d Cm \n", als);
                    printf("Tinggi = %d Cm \n", st);
                    printf("Keliling = %d Cm\n", Kel);
                    printf("Luas = %d Cm^2 \n", L);
    }